src/KafkaFlow.Serializer.ProtoBuf/KafkaFlow.Serializer.ProtoBuf.csproj

```diff
@@ -9,10 +9,14 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Confluent.Kafka" Version="1.6.3" />
+    <PackageReference Include="Confluent.SchemaRegistry" Version="1.6.3" />
+    <PackageReference Include="Confluent.SchemaRegistry.Serdes.Protobuf" Version="1.6.3" />
     <PackageReference Include="protobuf-net" Version="2.4.6" />
   </ItemGroup>
 
   <ItemGroup>
     <ProjectReference Include="..\KafkaFlow.Abstractions\KafkaFlow.Abstractions.csproj" />
+    <ProjectReference Include="..\KafkaFlow.Serializer\KafkaFlow.Serializer.csproj" />
```

Comment:
Please, undo the changes in this file

Reply:
Please see [this](https://github.com/Farfetch/kafka-flow/pull/140/files#r615706263).